app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1934,3 +1913,38 @@ h1.admin-welcome {
   font-size: 24px;
   color: white;
 }
+
+.order-id {
+  text-align: center;
+  margin: 50px 0px 10px 0px;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 40px;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.